app/styles/home.scss

```diff
@@ -111,7 +111,7 @@ button.small {
         line-height: 20px;
     }
 
-    > div { margin: 0 15px; }
+    > section { margin: 0 15px; }
```

Comment:
I think the [`div` on line 146 of this file](https://github.com/rust-lang/crates.io/pull/1227/commits/bd4c191c8a00b647f0683ed5ebe3ad4e496c66a7#diff-92c0575c9bf034586a7ce42ccddc91c4R146) needs to be updated to a `section` as well, otherwise the columns on the home page take up a variable amount of width when they should be taking up ~1/3 of the width, which starts happening at this commit.